src/Data/LinkHeaders.php

```diff
@@ -96,14 +96,14 @@ public function addFromString(string $link): static
         return $this;
     }
 
-    public function makeUnique()
+    public function makeUnique(): static
     {
         $handledHashes = [];
 
         foreach ($this->getLinkProvider()->getLinks() as $link) {
             /** @var Link $link */
-            $hash = md5($link->getHref(), serialize($link->getRels()));
-            if (! in_array($hash, $handledHashes)) {
+            $hash = md5($link->getHref(), true); // Previous the second parameter was: serialize($link->getRels()) which gives a string, where the second parameter excepts a boolean.
```

Comment:
You're right! This is what it should've been, good catch 🙂 
```suggestion
            $hash = md5($link->getHref() . serialize($link->getRels()));
```